python_modules/dagster/dagster/_core/storage/runs/schema.py

```diff
@@ -121,6 +121,22 @@
     db.Column("body", db.Text),
     db.Column("action_type", db.String(32)),
     db.Column("selector_id", db.Text),
+    db.Column("job_name", db.Text, nullable=True),
+)
+
+BackfillTagsTable = db.Table(
+    "backfill_tags",
+    RunStorageSqlMetadata,
+    db.Column(
+        "id",
+        db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+        primary_key=True,
+        autoincrement=True,
+    ),
+    db.Column("backfill_id", db.String(255)),
```

Comment:
depending on how you do the join / intersection, you will also want an index on `backfill_id` OR `bulk_actions_storage_id`.

See https://github.com/dagster-io/dagster/pull/22833 for example of why this makes a difference for run tag filtering.

Reply:
i think it'll need to be on `backfill_id`. I ended up having to remove the `bulk_action_storage_id` because the storage layer threw errors when i tried to get the result back from adding the backfill. seems like sqlite disallows getting the result of an insert call perhaps? i didn't spend much time investigating, but will do a bit of research to confirm that's the case before making this index

Reply:
ok nvm i just missed a part of the sql statement i needed to return the row id. all fixed now and i'll add the index

Reply:
you won't need the index if it's on storage id since that's the primary key

Reply:
i think i need to go back to backfill_id rather than storage id. the older versions of sqlalchemy all throw errors on the RETURNING statement to get the storage id when i add a backfill